src/views/Home.js

```diff
@@ -1,11 +1,41 @@
 import React, { Component } from "react";
+import Confirm from "../components/common/Buttons/confirm";
+import Input from "../components/common/Inputs/input";
+import Navbar from "../components/common/navbar";
+import MainCard from "../components/common/Cards/main";
+import Latest from "../components/common/Cards/latest";
 
-export default class Home extends Component {
+class Home extends Component {
   render() {
     return (
       <div>
-        <h2 className="home-container">Hello world, from Titan-Devs</h2>
+        <Navbar />
+        <div style={{ margin: 3 }}>
+          <h2 className="home-container">Hello world, from Titan-Devs</h2>
+          <Confirm
+            style={{ width: "40%" }}
```

Comment:
Bro, can you tell me why we are using inline CSS here?

Reply:
Good question @espoirMur, the reason for inline styles was to test if the reusable components can be called and tweaked by the component that called it. It was just to test if the called component will accept the new styles, they should be removed in the future.